### DISCUSSION TITLE
Can't find app/config/parameters.yml on Linux

### DISCUSSION THREAD

**Discussion by ReinerSanders** [wallabag/wallabag#7783] [General]
Hi,

I would love to count with your support please.

I installed Wallabag on Linux (EndeavourOS), using Docker. I created a custom path, everything is OK, but I can't find the `parameters.yml` file. I noticed the configutation mentioning it on `app/config/parameters.yml`, but, how do I access that?

I mean, `app` relative to what path?

Thanks in advance.

**Comment by nicosomb** [comment]
Hello @ReinerSanders. 
You have to access into your docker container. Then, in the root folder, there is a `app` folder.

**Comment by ReinerSanders** [comment]
Oh, that's great, thank you so much.

**Comment by HolgerAusB** [comment]
mhh, I believe it's `/var/www/wallabag/app/config/` within the container. I am not sure, but it could be possible that starting a command line will start start the prompt at `/var/www/wallabag`, so Nicolas is right too ;-)
